internal/storage/level_cache.go

```diff
@@ -0,0 +1,142 @@
+package storage
+
+import (
+	"go.uber.org/zap"
+	"golang.org/x/exp/mmap"
+
+	"github.com/milvus-io/milvus/internal/log"
+	"github.com/milvus-io/milvus/internal/util/cache"
+)
+
+type LevelCache struct {
+	memCache         *cache.LRU
+	localFileCache   *cache.LRU
+	localCacheEnable bool
+
+	memoryHitsCount     int
+	localCacheHitsCount int
+}
+
+func (l *LevelCache) Add(key cache.Key, value cache.Value) {
+	if l.localCacheEnable {
```

Comment:
If it's level cache, why does filecache and memcache has same value.
File cache should be filled only when memory cache evict